test/user.test.js

```diff
@@ -8,14 +8,15 @@ describe("testing all routes", () => {
     expect(response.statusCode).toBe(200);
   });
 });
-// testing adding a user endpoint
-describe("tesing user", ()=>{
-    test("adding a user", async () =>{
-        const response = await request(app).post("/signup").send({
-			firstName: "umurungi",
-			lastName: "helen",
-			email: "user@example.com",
+
+describe("vendor", ()=> {
```

Comment:
- test for post vendor is failing
- make sure you test more scenarios not just `post` successful. test failing scenarios too (validation, badrequest, authorization) 
- improve description to howto manually test this. specifically image type to submit.